reference_operation/include/reference_conv_fwd.hpp

```diff
@@ -60,7 +78,65 @@ struct ReferenceConvFwd : public device::BaseOperator
     };
 
     // Invoker
+    template <ck::index_t NDim>
```

Comment:
NDim is not necessary as template argument

Also "Invoker" should not have its own template argument, to be consistent with all DeviceOp::Invoker